packages/invoices/src/OutboundShipment/api/hooks/index.ts

```diff
@@ -1,6 +1,6 @@
-import * as Utils from './utils';
-import * as Lines from './line';
-import * as Document from './document';
+import { Utils } from './utils';
```

Comment:
made them consistent. removed the individual hook exports for Outbounds too - turns out we didn't use them